src/kw_include.sh

```diff
@@ -1,4 +1,27 @@
 #Used for file sourcing.
+
+#This function should used for getting variable names
+#
+# @fullpath Path to the file to be sourced
+#
+# Returns:
+# The expected varname for a succesful import, 22 if it's an invalid name
+function get_varname()
+{
+  local fullpath="$1"
+  local varname
+
+  varname="${fullpath#"$KW_LIB_DIR/"}" # leave path until KW_LIB_DIR
+  varname="${varname//\//_}"           # change bars to underlines
+  varname="${varname//\./_}"           # change dots into underscores
+  varname="${varname// /_}"            # change spaces into underscores
+  varname="${varname//-/_}"            # change dashes into underscores
```

Comment:
I wonder, can't we do all of this at once? Maybe with `sed`, I think it could look something like this: `varname="$(printf '%s\n' "$varname" | sed -e 's/[/.[:space:]-]/_/g')"`
or even: `varname="$(printf '%s\n' "$varname" | sed -e 's/[^[:alnum:]_]/_/g')"`
I think I prefer my second suggestion, as it will replace any characters that are not allowed as variable names to underscores. The only potential problem would be a `varname` starting with a number, but we don't currently deal with that.

Reply:
This is a really good suggestion, though if we are removing varname I don't see the point in keeping this function anyway.